File: FizzBuzzBizz.cs
public class FizzBuzzBizz{
	
	static void Main(){
		for(int i=1; i<=100; i++){
			if(i%105 == 0){
				System.Console.WriteLine("FizzBuzzBizz");
			} else if(i%35 == 0){
				System.Console.WriteLine("BuzzBizz");
			} else if(i%21 == 0){
				System.Console.WriteLine("FizzBizz");
			} else if(i%15 == 0){
				System.Console.WriteLine("FizzBuzz");
			} else if(i%7 == 0){
				System.Console.WriteLine("Bizz");
			} else if(i%5 == 0){
				System.Console.WriteLine("Buzz");
			} else if(i%3 == 0){
				System.Console.WriteLine("Fizz");
			} else {
				System.Console.WriteLine(i);
			}
		}

	}
}

File: FizzBuzzBizz.scala
object FizzBuzzBizz {
    def fizzBuzzBizz(n : Int) : String =
      (n % 7, n % 5, n % 3) match {
        case (0, 0, 0)  => s"FizzBuzzBizz $n"
        case (0, 0, _)  => s"FizzBuzz $n"
        case (0, _, 0)  => s"FizzBizz $n"
        case (0, _, _)  => s"Fizz $n"
        case (_, 0, 0)  => s"BuzzBizz $n"
        case (_, 0, _)  => s"Buzz $n"
        case (_, _, 0)  => s"Bizz $n"
        case _          => n.toString()
      }
}
